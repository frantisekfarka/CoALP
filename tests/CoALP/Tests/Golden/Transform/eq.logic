eq(pair(A,B)) :- eq(A), eq(B).
eq(Char).

