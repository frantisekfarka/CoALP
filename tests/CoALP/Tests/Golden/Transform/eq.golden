eq(pair(V_2i, V_3i), transform-func-1(V_5i, V_6i)) :- eq(V_2i, V_5i), eq(V_3i, V_6i).
eq(V_4i, transform-func-2).
